types/rockset/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": { "no-empty-interface": false  }
```

Comment:
Is there a reason why you need this off?

Reply:
I had few interfaces which are empty and are used in other interfaces. e.g. `AddDocumentsRequestData`. Is there any other way to resolve this?

Reply:
The linter documentation states: `An empty interface is equivalent to its supertype (or {}).`

So basically it serves no purpose other than saying the type is an `object`. Instead you could just use:
```ts
export interface AddDocumentsResponse {
    data: {}[];
}
```

Reply:
done